packages/training/python/src/training/action_executor.py

```
@@ -0,0 +1,621 @@
+            market_id=market_id,
+            side=side,
+            size=amount,
+            entry_price=market.yes_price if side == "yes" else market.no_price,
+            new_balance=self.portfolio.balance,
+            new_positions=self.portfolio.position_count,
+        )
+    
+    def _execute_prediction_sell(self, action: Dict) -> ActionResult:
+        """Execute prediction market sell"""
+        market_id = action["market"]
+        amount = action["amount"]
```

Ruff (0.14.10)

Local variable `amount` is assigned to but never used

Remove assignment to unused variable `amount`

(F841)